### ISSUE TITLE
Passthrough Http Auth bug (introduced in enterprise 1.9.0-beta3)

### ISSUE THREAD

**Issue by saiskee** [solo-io/gloo#5176]
**Describe the bug**
When using the passthrough http auth feature, auth configs report an error with the following warning: 
```
1 error occurred:                                                     
          * Unknown passthrough protocol type for name:"passthrough-auth"  name
space:"gloo-system"
```

**Comment by sam-heilbron** [comment]
@saiskee is this still an open bug or was this resolved?

**Comment by saiskee** [comment]
Fixed in 1.9.0-beta4; closing
